include/alpaka/kernel/TaskKernelOmp5.hpp

```diff
@@ -205,7 +205,7 @@ namespace alpaka
                     }
 
                     // After a block has been processed, the shared memory has to be deleted.
-                    // freeMem(acc);
+                    // freeSharedVars(acc);
```

Comment:
Why the comment? Can this be removed?

Reply:
It is not required in this backend and thus could be removed. The trait `freeSharedVars` would do nothing here. The same is true for `TaskKernelCpuOmp2Blocks`, `TaskKernelCpuSerial`, `TaskKernelCpuTbbBlocks` and `TaskKernelOacc`.